proposals/lip-0053.md

```diff
@@ -200,15 +200,26 @@ def verifyValidatorsUpdate(ccu: CCU) -> None:
     if bin(int.from_bytes(bftWeightsUpdateBitmap, 'big')).count("1") != len(bftWeightsUpdate):
         raise Exception("The number of 1s in the bitmap is not equal to the number of new BFT weights.")
 
-    # All new keys must have a positive BFT weight.
+    newValidatorsIndices = []                 // indices of bftWeightsUpdate array corresponding to new validators
+    index = 0
```

Comment:
Maybe it becomes easier to read if we rename this one to `indexBftWeightsUpdate`?

Reply:
also we should use python style comments

Reply:
Also update the comment to be in python style

Reply:
This array is now completely removed, so no need to name it ;-)